template/template-v1.json

```diff
@@ -279,6 +297,97 @@
         }
       }
     },
+    "CDKBootstrapPolicy": {
```

Comment:
This is a separate policy to allow the role to launch bootstrap cdk stack. In order to deploy EKS cluster in cloudformation, the boostrap cdk stack has to be created first.